packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -193,7 +200,7 @@ class DropdownButton extends React.Component {
           aria-disabled={isDisabled}
           ref={this.setButtonNode}
           aria-expanded={isOpen}
-          aria-label={selectText ? `${selectText}, ${selectedLabel}` : ''}
+          aria-label={selectText ? `${selectText}, ${selectedLabel}, ${label}, ${buttonAriaLabel}` : `${label}, ${buttonAriaLabel}`}
```

Comment:
what value does `label` has here ?? looks like it was not present on previous implementation

Reply:
Label has the button name. Earlier it was getting announced but with new changes its getting overridden. That's why I have concatenated in aria-label.

Reply:
have we verified with VO and jaws to ensure button label is not announced twice ?

Reply:
Yes